package.json

```diff
@@ -26,15 +26,16 @@
     "update:pkgs": "npx npm-check-updates -u"
   },
   "dependencies": {
-    "lunr-languages": "^1.9.0"
+    "lunr-languages": "^1.14.0"
```

Comment:
I think this line can be removed as we don't use lunr for search.

Reply:
Thanks, fixed by https://github.com/cncf/glossary/pull/3210/commits/557049345211cdf661c56b54360810b0530d08bc